src/main/java/bio/overture/ego/controller/ApiKeyController.java

```diff
@@ -138,27 +138,27 @@ public TokenController(
     val scopeNames = mapToList(scopes, ScopeName::new);
     val t = tokenService.issueToken(userId, scopeNames, description);
```

Comment:
we can safely rename `issueToken` from `TokenService.java` to `issueApiKey`, this will make the future refactor easier.